# request
Point[] point_array
---
# response
Point[] point_array
